dataprep/eda/basic/render.py

```diff
@@ -1176,11 +1176,11 @@ def render_basic(
                 False,
             )
             figs.append(fig)
-        elif dtype == DType.Numerical:
+        elif is_dtype(dtype, Continuous()):
             df, miss_pct = data
             fig = hist_viz(df, miss_pct, col, yscale, plot_width, plot_height, False)
             figs.append(fig)
-        elif dtype == DType.DateTime:
+        elif dtype == DateTime:
```

Comment:
Should use `is_dtype(dtype, DateTime())`?

Reply:
Yeah, you are right, I will fix these.